### ISSUE TITLE
Error 'SyntaxError: invalid syntax' was encountered in python3.10

### ISSUE THREAD

**Issue by hezhefly** [pypae/pydantic-typer#9]
```bash
File "/data/heweicheng/miniconda3/envs/xinference/lib/python3.10/site-packages/pydantic_typer/main.py", line 156
    return origin[*updated_args]
                  ^
SyntaxError: invalid syntax
```

**Comment by pypae** [comment]
Thanks for reporting. Seems like this syntax was introduced in Python 3.11 with [PEP 646](https://peps.python.org/pep-0646/#unpack-for-backwards-compatibility). I just pushed a fix and released version [`0.0.12`](https://pypi.org/project/pydantic-typer/0.0.12/).

**Comment by pypae** [comment]
Seems like the tests are still failing, I'm having another look later.

**Comment by pypae** [comment]
It's fixed in [`0.0.13`](https://pypi.org/project/pydantic-typer/0.0.13/)
